musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/instrument/noteSheet/NoteSheetView.java

```diff
@@ -26,29 +26,43 @@
 import android.graphics.Canvas;
 import android.util.AttributeSet;
 import android.util.DisplayMetrics;
+import android.view.MotionEvent;
 import android.view.View;
 import android.view.WindowManager;
 
 import org.catrobat.musicdroid.pocketmusic.instrument.piano.PianoActivity;
-import org.catrobat.musicdroid.pocketmusic.note.MusicalInstrument;
 import org.catrobat.musicdroid.pocketmusic.note.MusicalKey;
-import org.catrobat.musicdroid.pocketmusic.note.Project;
 import org.catrobat.musicdroid.pocketmusic.note.Track;
+import org.catrobat.musicdroid.pocketmusic.note.draw.DrawElementsTouchDetector;
 import org.catrobat.musicdroid.pocketmusic.note.draw.NoteSheetCanvas;
 import org.catrobat.musicdroid.pocketmusic.note.draw.NoteSheetDrawer;
+import org.catrobat.musicdroid.pocketmusic.note.draw.SymbolPosition;
+import org.catrobat.musicdroid.pocketmusic.note.symbol.Symbol;
+import org.catrobat.musicdroid.pocketmusic.note.symbol.TrackToSymbolsConverter;
+
+import java.util.LinkedList;
+import java.util.List;
 
 public class NoteSheetView extends View {
 
-	private Track track;
+    protected DrawElementsTouchDetector touchDetector;
+    protected TrackToSymbolsConverter trackConverter;
+    protected List<Symbol> symbols;
+    protected MusicalKey key;
+    protected List<SymbolPosition> symbolPositions;
```

Comment:
The two lists "symbols" and "symbolPositions" have - as far as I understand - always to be synchronous. I think we should create a new data-structure for that so that this is ensured.